cxx-qt-gen/test_outputs/invokables.h

```diff
@@ -36,6 +36,11 @@ class MyObject : public QObject
   Q_INVOKABLE qint32 invokableReturnPrimitive();
   Q_INVOKABLE QString invokableReturnStatic();
 
+  void cppOnlyMethod();
```

Comment:
These 4 methods should actually not be generated into C++.
They just need to be in the `impl cxx_qt::QObject<T>` block because that's the only place, where, in future, you'll still have access to C++ functionality (e.g. signals, etc.).

Reply:
Eventually yes, when we move the implementation of the invokables onto the C++ object. For now we still have the proxies.

Reply:
Removed :-)